src/plugins/intel_cpu/src/node.cpp

```diff
@@ -562,18 +566,22 @@ void Node::updateDynamicParams() {
                     getTypeStr(),
                     " with name: ",
                     getName());
-    if (isExecutable()) {
-        if (needPrepareParams()) {
-            OPENVINO_ASSERT(inputShapesDefined(),
-                            "Can't prepare params for ",
-                            getTypeStr(),
-                            " node with name: ",
-                            getName(),
-                            " since the input shapes are not defined.");
-            DEBUG_LOG(" prepareParams() on #", getExecIndex(), " ", getTypeStr(), " ", algToString(getAlgorithm()),
-                      " ", getName(), " ", getOriginalLayers());
-            prepareParams();
+    try {
+        if (isExecutable()) {
+            if (needPrepareParams()) {
+                OPENVINO_ASSERT(inputShapesDefined(),
+                                "Can't prepare params for ",
+                                getTypeStr(),
+                                " node with name: ",
+                                getName(),
+                                " since the input shapes are not defined.");
+                DEBUG_LOG(" prepareParams() on #", getExecIndex(), " ", getTypeStr(), " ", algToString(getAlgorithm()),
+                        " ", getName(), " ", getOriginalLayers());
+                prepareParams();
+            }
         }
+    } catch (const std::exception& e) {
+        THROW_CPU_NODE_ERR(e.what());
```

Comment:
Just curious, why in some blocks only `std::runtime_error` is caught, while here `std::exception` is used as the type to catch?